airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -2125,25 +2155,42 @@ abstract class BasicFunctionalityIntegrationTest(
                         "boolean" to null,
                         "timestamp_with_timezone" to null,
                         "timestamp_without_timezone" to null,
-                        "time_with_timezone" to null,
+                        "time_with_timezone" to
+                            timetz(
+                                "\"foo\"",
+                                null,
+                            ),
                         "time_without_timezone" to null,
                         "date" to null,
                     )
                 badValuesChanges =
                     (stream.schema as ObjectType)
                         .properties
                         .keys
+                        .asSequence()
```

Comment:
intellij got angry that I was chaining so many filter/map operations and told me to do this :P